index.js

```diff
@@ -1,54 +1,61 @@
 const core = require('@actions/core');
-const shell = require('shelljs');
-const { readdirSync, rename } = require('fs');
-
-const AUDIT_COMMAND = core.getInput('AUDIT_COMMAND');
-const JSON_DRILLER = core.getInput('JSON_DRILLER');
-
-const cleanUpEnvironment = () => {
-  // change the directory in order to properly run the audit command
-  shell.cd(`${process.env.GITHUB_WORKSPACE}/${process.env.ACTION_NAME}`);
-
-  const files = readdirSync(__dirname);
-
-  // rename the audit-specific files
-  const nonRootFilteredFiles = files.filter(
-    (file) => file.includes('package') && !file.includes('root')
-  );
-  nonRootFilteredFiles.forEach((file) => rename(file, `${file}_audit`, (err) => console.log(err)));
-
-  // rename the root related packages
-  console.log('Renaming the root related package files...');
-  const rootFilteredFiles = files.filter(
-    (file) => file.includes('package') && file.includes('root')
-  );
-
-  rootFilteredFiles.forEach((file) =>
-    rename(file, file.replace('-root', ''), (err) => console.log(err))
-  );
-};
-
-const actionRunner = (AUDIT_COMMAND) => {
-  console.log('Prepare the container for the audit...');
-  cleanUpEnvironment();
-
-  console.log(`Attempting to execute ${AUDIT_COMMAND}...`);
-  let auditCommandOutput = '';
-  const auditCommand = AUDIT_COMMAND;
-
-  const { stdout, stderr } = shell.exec(auditCommand);
-
-  if (stdout) {
-    console.log('Command executed successfully!');
-    auditCommandOutput = stdout;
-  } else if (stderr) {
-    console.log(`Command execution encountered the following error: ${stderr}`);
-    auditCommandOutput = stderr;
+const fs = require('fs');
+const { v4 } = require('uuid');
+const jsonXform = require('@perpk/json-xform');
+const bunyan = require('bunyan');
+const log = bunyan.createLogger({ name: 'actions-audit' });
+const { preprocessOwaspReport } = require('./utils/helpers');
+const REPORT_INPUT = core.getInput('REPORT_INPUT') || process.env.REPORT_INPUT;
+const AUDIT_TOOL = core.getInput('AUDIT_TOOL') || process.env.AUDIT_TOOL;
+let auditReportFlattened;
+
+const tempReportInputFile = fs.readFileSync(REPORT_INPUT, 'utf8');
```

Comment:
As a general comment, it would be nice to structure the deps and consts as follows:

- npm depdencies
- local dependencies
- constants
- variables

all of the separated by new line per category.

Reply:
The structure has been changed based on the above suggestion.